src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,78 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService =
+                (DriverService) injector.getInstance(DriverService.class);
+        Driver driver = new Driver();
+        driver.setName("Michel Rat");
+        driver.setLicenseNumber("8-800-555-355");
+        driverService.create(driver);
+
+        driver.setLicenseNumber("345-66-5-345");
+        driver.setName("Igor Terech");
+        driverService.create(driver);
+
+        driver.setLicenseNumber("65-566-555-5-315");
+        driver.setName("Taras Boch");
+        driverService.create(driver);
+
+        driver.setLicenseNumber("45-664-555-66");
+        driver.setName("Taras Boch");
+        driverService.create(driver);
+
+        driverService.delete(1L);
+        driver.setName("Mick Rock");
+        driver.setId(5L);
+        driverService.update(driver);
+        System.out.println(driverService.get(5L) + "\n");
+        driverService.getAll().stream().forEach(System.out::println);
```

Comment:
```suggestion
        Driver driverMichel = new Driver();
        driver.setName("Michel Rat");
        driver.setLicenseNumber("8-800-555-355");
        driverService.create(driver);

	Driver driverIigor = new Driver();
        driver.setLicenseNumber("345-66-5-345");
        driver.setName("Igor Terech");
        driverService.create(driver);
         
         .....
         
        driverService.delete(driverMichel.getId());
        driverIigor.setName("Mick Rock");
        driverService.update(driverIigor);
        System.out.println(driverService.get(driverIigor.getId()) + "\n");
        driverService.getAll().stream().forEach(System.out::println);
```

It's better not to use Id explicitly as they could change
Same in other places